src/scripts.js

```diff
@@ -76,121 +75,138 @@ function getRecipes() {
 function loadPageInfo(allData) {
   let allUsersData = allData[0];
   ingredientsData = allData[1];
-  recipeData = allData[2];
-  currentUser = new User(allUsersData[Math.floor(Math.random()
-    * allUsersData.length)]);
+  recipeData = instantiateRecipes(allData[2]);
+  currentUser = new User(allUsersData[Math.floor(Math.random() * allUsersData.length)]);
   domUpdates.welcomeUser(currentUser);
   findPantryInfo(ingredientsData);
-  findTags(recipeData);
-  createCards(recipeData);
+  createRecipeTypeList(recipeData);
+  createRecipeCards(recipeData);
 }
 
-// CREATE RECIPE CARDS
-function createCards(recipeData) {
+function updatePantryIngredients(ingredID, ingredMod) {
+  fetch('https://fe-apps.herokuapp.com/api/v1/whats-cookin/1911/users/wcUsersData', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({
+      "userID": currentUser.id,
+      "ingredientID": +ingredID,
+      "ingredientModification": +ingredMod
+    })
+  })
+    .then(response => response.json())
+    .then(data => console.log(data))
+    .catch(error => console.log(error))
+}
+
+// ************ RECIPES *************** //
+function instantiateRecipes(recipes) {
+  recipes = recipes.map(recipe => new Recipe(recipe));
+  return recipes
+}
+
```

Comment:
I like how you moved this Fetch post request up to display when it is being called. It makes more sense this way

Reply:
I like the new variable names you used. Demonstrates more developer empathy.